### PR TITLE
Update player-vehicle.pwn

### PR DESCRIPTION
**Pull request by kolor4do**

### CODE REVIEW

**Comment by Southclaws** [comment]
Could you describe what this change fixes please? Thanks!

**Comment by nicolasmenegoni** [comment]
If you save a vehicle and immediately after that another player (who does not have a vehicle) enters it, the vehicle will be doubled after respawn.

Not only that, sometimes someone's vehicle disappeared mysteriously. I believe that this way you will be saved correctly. At least that's what I tested.

**Comment by nicolasmenegoni** [comment]
> 
> 
> Could you describe what this change fixes please? Thanks!

If you save a vehicle and another player (who has no vehicle) enters it, the vehicle will be doubled after respawn.

Not only that, sometimes someone's vehicle disappeared mysteriously. I believe that this way you will be saved correctly. At least that's what I tested.

File: gamemodes/sss/core/vehicle/player-vehicle.pwn [hunk 1]
```diff
@@ -772,75 +776,31 @@ _UpdatePlayerVehicle(playerid, vehicleid)
...
 		else
 		{
-			// Vehicle's owner is not the player, check if the player already
-			// owns a vehicle
-			dbg("player-vehicle", 1, "[_UpdatePlayerVehicle] This vehicle does not belong to the player in context, swapping");
-
 			if(pveh_PlayerVehicle[playerid] != INVALID_VEHICLE_ID)
 			{
-				// Player owns a vehicle
-				dbg("player-vehicle", 1, "[_UpdatePlayerVehicle] Player in context already owns vehicle %s (%d) swapping owners", GetVehicleUUID(pveh_PlayerVehicle[playerid]), pveh_PlayerVehicle[playerid]);
```
Comment (Southclaws) [inline gamemodes/sss/core/vehicle/player-vehicle.pwn:812]: Ownership swapping was done to allow members of the same team to drive each other's vehicles, for example if you're in a base and moving vehicles around. I think removing this feature will result in unexpected vehicle loss.

**Pull request closed without merging**